{{ cookiecutter.app_name }}/tests/{{ cookiecutter.module_name }}.py

```diff
@@ -0,0 +1,86 @@
+{%- if cookiecutter.test_framework == 'pytest' %}
+import os
+import sys
+import tempfile
```

Comment:
There's a lot of identical code between the pytest and the unittest versions: factoring it out would make both the similarities and the differences easier to identify.

Reply:
So - I tried doing that originally. There's definitely lots of shared code (although most of that code will disappear once we get rid of the Android content-extraction hack); however, I ended up coming to the opinion that the duplication was better than the alternative. In order to preserve black-compatible import ordering and line spacing, you end up dropping a lot of if-elif blocks, and the end result ends up being harder to read - and even harder to get right preserving black-compatible spacing.